python/paddle/vision/ops.py

```diff
@@ -1540,3 +1530,146 @@ def _nms(boxes, iou_threshold):
         return keep_boxes_idxs[topk_sub_indices]
 
     return keep_boxes_idxs[sorted_sub_indices][:top_k]
+
+
+def generate_proposals(scores,
```

Comment:
Done. 
Through the discusstion with @zhiboniu , current now we change implement of the corresponding fluid api to this `paddle.vision.ops.generate_proposals`. And we still keep this fluid api to avoid potential incompatibilities.